sdk/internal/temporal/resource.go

```diff
@@ -91,7 +119,6 @@ func (er *Resource[TResource, TState]) Get(state TState) (TResource, error) {
 		// This thread/goroutine has been selected to acquire/update the resource
 		var expiration time.Time
 		var newValue TResource
-		er.lastAttempt = now
```

Comment:
Is moving this necessary and safe? This will change one of the conditions when it's set.

Reply:
I believe it's safe, however it isn't necessary or worth the churn. I reverted the move.